site/content/config.js

```diff
@@ -10,7 +9,11 @@ const config = {
     { href: 'https://github.com/flowershow/flowershow/discussions', name: 'Forum' },
   ],
   nextSeo: {
+    title: 'Flowershow',
+    description: 'Turn your markdown notes into an elegant website and tailor it to your needs. Flowershow is easy to use, fully-featured, Obsidian compatible and open-source.',
+    canonical: 'https://previews-823b8d.netlify.live',
```

Comment:
This seems an odd canonical url - surely it's https://flowershow.app/

Reply:
Yes! Forgot to change that as I was testing. Thanks for pointing out !!